examples/vision/yolov8.py

```diff
@@ -0,0 +1,628 @@
+"""
+Title: Efficient Object Detection with YOLOV8 and KerasCV
```

Comment:
The title sounds more like a guide. 
How about using a more task-oriented title?

Reply:
@innat-asj can u suggest a title?

Reply:
It depends on your task. Though you're using the SDC dataset, I think it doesn't bring multi-level complexities (multi-modal). Instead, it's simple multi-class, isn't it? So, I think you can simply go with mult-class object detection using yolov8 or something.